### ISSUE TITLE
need to catch level>27 and level <0

### ISSUE THREAD

**Issue by NiklasPhabian** [SpatioTemporal/pystare#35]
pystare happily eats levels < 0 and >27 and produces seemingly plausible SIDs. We need to catch this as invalid input!

```python
>>> import pystare
>>> x = [-119.42]
>>> y = [34.25]

>>> pystare.from_latlon(y, x, 27)
array([3331752989521980123)

>>>  pystare.from_latlon(y, x, 50)
array([3331752989521980146])

>>>  pystare.from_latlon(y, x, -1)
array([4611686018427387903])

**Comment by NiklasPhabian** [comment]
Thankfully, those are just a handful of functions. 
All of them addressed in https://github.com/SpatioTemporal/pystare/commit/22f02d12f0f0e1771807a047851c622ac8c72174

**Comment by NiklasPhabian** [comment]
OK. IMO, this is solved in #61
